app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class QuestionsController < ApplicationController
+    before_action :authenticate_user!, only: [:new, :create,:idnex,:show]
+
+    def show
+        @question = Question.find(params[:id])
+        @answer = Answer.new
+    end
+
+    def index
+        @questions = Questions.all.order(id: :asc)
+        @question = Question.new
+    end
+
+    def create
+        @question = Question.new(question_params)
+        if @question.save
+            flash[:notice] = "質問を投稿しました。"
+            redirect_to("/questions/index")
```

Comment:
`redirect_to`の場合はこの書き方ができなかったと思います。

```suggestion
            redirect_to action: :index
```